docs/src/components/home/Hero.tsx

```diff
@@ -110,7 +110,7 @@ export default function Hero() {
       left={
         <Box sx={{ textAlign: { xs: 'center', md: 'left' } }}>
           <Typography variant="h1" sx={{ my: 2, maxWidth: 500 }}>
-            The <GradientText>ultimate</GradientText> solution for your&nbsp;UI
+            The <GradientText>ultimate</GradientText> solution for building&nbsp;UIs
```

Comment:
@mbrookes What do you think about removing _"your"_ and adding a plural to UI? Is it weird?

Reply:
A bit...

Reply:
The downside of keeping _"your"_ while adding _"building"_ is that we get an extra line just for _"building"_ at large viewports (most of our visitors). For comparison:

| With _"your"_ | Without it |
| ----------- | ----------- |
| <img width="700" alt="Screen Shot 2021-09-01 at 19 06 01" src="https://user-images.githubusercontent.com/67129314/131751675-c0d383a7-5438-4e27-a194-37587c1c854f.png"> | <img width="700" alt="Screen Shot 2021-09-01 at 19 06 22" src="https://user-images.githubusercontent.com/67129314/131751689-2334b30d-954c-45e4-b4dc-455ace15e440.png"> |

I'd personally be okay with moving forward without _"your"_ and with _"UIs"_.

Reply:
I like the new one, without "your"

Reply:
@oliviertassinari @mbrookes 

Hey! Just for better organizing discussions — as I see you are reflecting upon the Hero tagline — we already have this opened conversation/thread around it we can use (so whenever we settle on something, we can then resolve it). 

> what does it even mean?

Let me then just share my thoughts around it: I wrote that wanting, yes, to be very bold. From personal experience and from friends and colleagues from multiple companies I know using the library, I wouldn't think it's that far ahead. In the last company I worked at, MUI enabled the rapid (6 month period) launch of 4 products since they all consumed the same DS, one of which we only had the job of tweaking the theme. So it was basically effortless UI-wise, in comparison to back-end-related tasks. And that was sort of water to wine transformation in that company, making front-end the least of our problems. Especially since we have had so many headaches trying to evolve and maintain an in-house built DS.

So, that's why "_ultimate solution for building UIs_". It's bold and ambitious and it sounds impacting. People have different experiences with the library (most might use it for back-office-related products or other types of "less shiny" projects) but I've seen it being used to support enormous companies, with lots of users and all sorts of use-cases/interactions. So I think it also works for promoting this capability. 

Happy to iterate, if you have any suggestions or options, let me know.

Reply:
I would propose we come up with different proposals, and have the team vote on the one that resonates the most.

### 🚀 Option A: _Current_
Resumed [why:](https://github.com/mui-org/material-ui/pull/28081#discussion_r701327496) "ultimate solution for building UIs"? It's bold and ambitious and it sounds impacting. People have different experiences with the library (most might use it for back-office-related products or other types of "less shiny" projects) but I've seen it being used to support enormous companies, with lots of users and all sorts of use-cases/interactions. So I think it also works for promoting this capability.

   - **SEO title:** "MUI: A popular React UI framework"
   - **SEO description:** "The ultimate solution for building UIs. MUI provides a robust, customizable, and accessible library of foundational and advanced components, enabling you to build your own design system and develop React applications faster."
   - **Page h1:** "The ultimate solution for building UIs"
   - **Page description:** "MUI provides a robust, customizable, and accessible library of foundational and advanced components, enabling you to build your own design system and develop React applications faster."

| Website Hero | Google Search |
| ----------- | ----------- |
| <img width="600" alt="Screen Shot 2021-09-02 at 17 21 27" src="https://user-images.githubusercontent.com/67129314/131911039-35628291-1e86-4256-9d96-d9aae7fdbab9.png"> | <img width="600" alt="Screen Shot 2021-09-02 at 17 28 19" src="https://user-images.githubusercontent.com/67129314/131911717-1f9367a8-18db-4158-9e48-c9f4dc1a5ad5.png"> |

⚠️ Attention to the description ellipse, we might want to shorten it.

### 🎉 Option B: _Core product focus_
It's skewed toward the core product, as a frequent entry point for using the library. With this option, I think that the release of the unstyled and joy components will warrant updating the headers. I also think that the release of a Studio would require updating the headers too.

   - **SEO title:** "MUI - A popular React UI component library"
   - **SEO description:** "MUI provides a simple, customizable, and accessible library of React components. Follow your own design system, or start with Material Design. You will develop React applications faster."
   - **Page h1:** "Quickly build beautiful React UIs"
   - **Page description:** "MUI provides a simple, customizable, and accessible library of React components. Follow your own design system, or start with Material Design. You will develop React applications faster."

| Website Hero | Google Search |
| ----------- | ----------- |
|<img width="600" alt="Screen Shot 2021-09-02 at 17 22 39" src="https://user-images.githubusercontent.com/67129314/131911052-3b09127f-dec5-48a9-9684-94ebc258f471.png"> | <img width="600" alt="Screenshot 2021-09-02 at 20 33 29" src="https://user-images.githubusercontent.com/3165635/131900012-73069469-2221-467c-914b-f1a179fed9f1.png">

### 👍 Option C: _[please add]_

---

I focused on the home, for the other product pages, assuming that the home would rank for existing SEO queries of the homepage (MUI Core), The copywriting on these pages looks great to me.

Reply:
> ⚠️ Attention to the description ellipse, we might want to shorten it.

I figure that removing the ellipses is rather simple. Option A is pasting the hero tagline on the SEO description while B isn't 😬 Just doing the same on A would probably work!

Reply:
@mbrookes If you could add Option C, the one you were considering, I think that it would be great. Regarding my vote, I would vote in this order:

- SEO title:  🎉 > 🚀
- SEO description:  🎉 > 🚀
- Page h1:  🚀 > 🎉 
- Page description: 🚀 > 🎉 

My flow of thoughts is: for the SEO entry point, get as specific as possible to what we have to offer, for the landing page, have a bold tagline, be inspirational, to what we are about.

Reply:
> A popular React UI component library

Perhaps we should brag a bit and change it to "The most popular React UI component library"?
As for other options, we could emphasize the size and completeness of the library: "The comprehensive React UI component toolkit"

> Quickly build beautiful React UIs

I feel like something is wrong with this sentence. I'd say "Build beautiful React UIs quickly" but maybe it's just me.

> MUI provides a simple, customizable, and accessible library of React components. Follow your own design system, or start with Material Design. You will develop React applications faster.

I like this one. I'd just remove the comma before "or start with Material Design"

As for "library" vs "framework" - I'm for the former. I once read the difference is "you call the library, the framework calls you".

Reply:
I have been reflecting on this for the last couple of days... I think that _"The ultimate solution for building UIs"_ is good but maybe not great indeed. There's something about the ultimate word that started bothering me, it can be confusing. And _"A popular React UI component library"_ also doesn't sound great. Popularity is a consequence of the value the product offers, definitely a signal of success but not necessarily. Nobody should choose anything focusing on popularity.

So, I went on a bit of research about the making/writing of value prop statements and found two interesting definitions of it that can help to generate better options:

> _"A value proposition is a quality of your product that is matched with a benefit."_
> _"In short, value props are the selling points that increase a visitor's conversion desire."_ 

Taking that into consideration, I quickly thought about three main qualities MUI as a product/library have, that weights in the decision of choosing it in a given application:
- Quantity of components and other resources (completeness)
- Amount of people contributing (safeness)
- Time of existence (maturity) 

I'm sure we can think of more and more. The thing is we can't use all of them since it's a short sentence. The one that resonates with me the most, especially given my prior experience with MUI before joining the team, is the first one. A quality should offer a benefit (outcome) for the user. Thinking about _completeness_, it quickly comes to mind that it can probably support most of my needs/use-cases. And this has been indeed my experience. At prior companies, MUI replaced a lot of scattered libs we used here and there, simplified the system, and lowered the maintenance cost. So here's another option that I believe follows the above rationale:

<img width="500" alt="Screen Shot 2021-09-09 at 19 08 04" src="https://user-images.githubusercontent.com/67129314/132768950-7a959322-b5cb-4f35-a4b0-73dd323f1156.png">

_"Everything"_ is meant to be bold. There isn't probably really everything but we're aiming towards that. _"...to build React UIs"_ bringing back this because not only it might help with SEO but it narrows the product space (the previous _"Ultimate solution for building UIs"_ only would work for those choosing to use React as the stack of choice. All people that don't wouldn't feel like the lib is the ultimate thing for UIs). And lastly, _"...faster"_ is the outcome: If everything's here, then there's less context switching, therefore a more rapid development.

So, in summary, maybe as an option C:

- **SEO title:** "MUI - A popular React UI component library"
- **SEO description:** "MUI provides a simple, customizable, and accessible library of React components. Follow your own design system, or start with Material Design. You will develop React applications faster."
- **Page h1:** "Everything to build React UIs faster"
- **Page description:** "MUI provides a robust, customizable, and accessible library of foundational and advanced components, enabling you to build your own design system and develop React applications faster."

I'm just not sure if it's a good or bad thing having SEO title/descriptions completely different from the page ones. Though I suppose that "component library"  and "Material Design", in the title and description respectively, might help to boost SEO. 

It would be awesome if we got to resolve this at maximum by next Monday, as we should have it concluded for the website release with v5 stable. Let me know what you think.

Reply:
@danilo-leal Do you think this can be resolved today?

Reply:
Option C sounds good to me, happy to move forward with it. A few thoughts:

1. Agree, not sure that "the most popular" really matters. Or to put it differently, that it's the best wait to improve the SEO click conversion. We have tried to remove the ego part of it in the past by saying "A popular" over "The most popular" like Bootstrap and MongoDB are doing.
2. One downside of "Everything". I think that as I developer, I would be worried if it was really the objective of the library I was picking. I would prefer the solution I pick to do one thing, really well. Maybe it's just me that saw Angular unbundled by React. At the same time, Kendo frames it very similarrly: "The only React component library you need."
3. SEO titles and page content are usually not completely aligned, but close. A look at some of the biggest open-source companies:

- https://www.hashicorp.com/:
  - SEO title: HashiCorp: Infrastructure enables innovation
  - SEO description: HashiCorp delivers consistent workflows to provision, secure, connect, and run any infrastructure for any application.
  - Page title: Fully managed infrastructure for your cloud operations  
- https://www.mongodb.com/
  - SEO title: MongoDB: The most popular database for modern apps
  - SEO description: We're the creators of MongoDB, the most popular database for modern apps, and MongoDB Atlas, the global cloud database on AWS, Azure, and GCP.
  - Page title: MongoDB The application data platform
  - Page description: Accelerate development, address diverse data sets, and adapt quickly to change with a proven application data platform built around the database most wanted by developers 4 years running.
- https://www.confluent.io/
  - SEO title: Confluent: Data in Motion
  - SEO description: Confluent is building the foundational platform for data in motion so any organization can innovate and win in a digital-first world.
  - Page title: Set Your Data in Motion
  - Page description: Your business is in motion. Now, you have a data platform for it. Confluent makes it easy to connect your apps, systems, and your entire organization with real-time data flow and processing.

Reply:
> "Everything" is meant to be bold. It probably doesn't have really everything but we're aiming towards that.

"Everything" can mean breadth of components, but could also be taken to mean a lot more. Things that MUI doesn't provide, being a library, not a framework. It is ambiguous, if not confusing.

> Kendo frames it very similarly: "The only React component library you need."

And hits two keywords that we're missing ("component library"), while skipping the awkward plural "UIs".

I'm going to put value statement creation up there with "naming things" in the difficulty spectrum! 😄

Reply:
"The React component library you always wanted" 😄

Reply:
How about _"Most things to build React UIs faster"_?! 🤣 Just kidding. But yeah, dang, this is hard. 

> And hits two keywords that we're missing ("component library"), while skipping the awkward plural "UIs".

Kendo's usage of "component library" is in the SEO title and page paragraph description (not home h1). We also have these two words being used in the same places, so it's probably fine. Agree though that _"everything"_ can be misleading, kinda hoped you'd ignore it 😅 

The tricky thing about the home page h1 for me I believe is that it should offer a benefit. It can but it doesn't need to be descriptive of what the product/company actually is. A few examples:

-  **Stitches**: _Style your components with confidence_ (No actual description of the product)
-  **Sketch**: _Your design system starts here_ (No actual description of the product)
- **Stripe**: _Payments infrastructure for the internet_ (Description of the product while smartly telling you the benefit - have payment infra)
- **[Tempo](https://www.yourtempo.co/)**: _The email client that helps you focus_ (Does both)

What if we go with something already used on our current home page (soon to be old-one). It seems to apply, be more accurate and grounded:

<img width="500" alt="Screen Shot 2021-09-13 at 20 08 58" src="https://user-images.githubusercontent.com/67129314/133168609-3d5c116f-7e4c-467f-a440-99af0ffa9b4b.png">

Reply:
"faster and _more_ beautiful..." to fix the grammar

"beautiful" is a bit of an odd word to use to describe a React app though.

"The React component library you always wanted"

Tells you what it is, indirectly gives a benefit (it solves the problems your current component library doesn't), with a nice warm feel to it.

Reply:
Does this work for you? Changed _"component"_ to UI because _"component library"_ will keep breaking into two lines. I plan on highlighting with the gradient the _"React UI library"_, make sense?

We can have _"The React component library you always wanted"_ just like that on the SEO title but for the home h1 it's too big.

| Full | "Simplified" |
| ----------- | ----------- |
| <img width="500" alt="Screen Shot 2021-09-14 at 09 13 31" src="https://user-images.githubusercontent.com/67129314/133260051-6bc09842-92c3-4327-826c-e952b35f564f.png"> | <img width="500" alt="Screen Shot 2021-09-14 at 09 45 11" src="https://user-images.githubusercontent.com/67129314/133260035-bcd3291b-8f4e-4a39-a5f0-4df481a6dfa9.png"> |

Reply:
I have mixed feelings about this one. "The React component library you always wanted" reads much better, but I understand it's too long. How about "The React UI library you need" instead?

Reply:
I believe it works too. I also questioned myself which verb would work better, between _want_ and _need_, for this "simplified" version... For some reason, using _want_ sounds better to me in the past tense. _Need_ might work fine in both present and past?! No big preference.

Reply:
> "The React UI library you need"

Is missing a word or two:

"The _only_ React UI _component_ library you need"

... which is Kendo's 😁 

Either we make more room for the current suggestion, or it's back to the drawing board... 😭

Reply:
"It's like Material-UI, but better"

:)

Reply:
"MUI. All style, all substance".

Reply:
Not sure if you're joking or not 😅 😅 😅 Though I think I'd be down with it 🤣

Reply:
Alright, reduced just a tad bit the upper bound h1 font size to fit the sentence better. But between these two, I think `UI library` distributes the word better... How are we?!

| Component library | UI library |
| ----------- | ----------- |
| <img width="500" alt="Screen Shot 2021-09-14 at 15 56 52" src="https://user-images.githubusercontent.com/67129314/133317779-1576cbbc-ed07-49d7-8df2-010432aba2c5.png"> | <img width="500" alt="Screen Shot 2021-09-14 at 15 57 20" src="https://user-images.githubusercontent.com/67129314/133317797-193add0f-7de9-4946-8214-ee13c118f475.png">  |

Reply:
@mbrookes Going once, going twice... can we wrap it with `UI library`? `Component library` is in the SEO title, we'll have both.

Reply:
Sold.

Reply:
🎉 and we can always change it